### DISCUSSION TITLE
Discord??

### DISCUSSION THREAD

**Discussion by NoelFB** [FosterFramework/Foster#20] [Ideas]
I know the actual amount of people using this right now is pretty small, but wondering if there's an interest in making a Discord? I like Github for more official/tracked discussions, but Discord might be nice for more casual conversations.

**Comment by MrBrixican** [comment]
Yeah, that would be great!

**Comment by NoelFB** [comment]
Created one... Will see how useful it is but I think it'll be convenient.
https://discord.gg/K7tdFuP3Bg
